src/Makefile

```diff
@@ -21,6 +21,21 @@ server:
 	mkdir -p build
 	cp $$GOPATH/bin/server build/server
 
+migrator-lean:
```

Comment:
Tiny, tiny naming nit -- the `lean` flag is an implementation detail. Can we call this `make migrator-release` and then add a target called `release` which is just a dummy target for all 3 of these?